content/lang/fr/about/demos-examples.md

```diff
@@ -1,97 +1,97 @@
 ---
-name: Demos & Examples
+name: Démos & Exemples
 permalink: '/about/demos-examples'
 ---
 
-# Demos & Examples
+# Démos & Exemples
 
-This pages lists a bunch of demos and examples you can use to learn Preact.
+Cette page liste des démos et exemples que vous pouvez utiliser pour apprendre Preact.
 
-> :information_desk_person: _Built one of your own?
-> [Add it!](https://github.com/developit/preact-www/blob/master/content/about/demos-examples.md)_
+> :information_desk_person: _Vous avez créé votre propre démo ?
+> [Ajoutez-le !](https://github.com/developit/preact-www/blob/master/content/about/demos-examples.md)_
 
 
-## Full Apps
+## Applications Complètes
 
-[**Preact Website** _(preactjs.com)_](https://preactjs.com)  
-Of course this website is built with Preact.  
-[Github Project](https://github.com/developit/preact-www)
+[**Site Web de Preact** _(preactjs.com)_](https://preactjs.com)  
+Ce site web a bien-entendu été créé avec Preact.  
+[Projet Github](https://github.com/developit/preact-www)
 
-**[ESBench](http://esbench.com)** :alarm_clock:  
-Built with Preact & Material Design Lite.
+**[ESBench](http://esbench.com)** :alarm_clock:
+Créé avec Preact & Material Design Lite.
 
 [**GuriVR**](https://gurivr.com) :eyeglasses:
 Natural language based Web VR story creator.  
```

Comment:
I think @developit’s google translation is OK here. `Créateur d'histoire Web VR basé sur le langage naturel.` (just added some accents).